recipes-extended/libtirpc/libtirpc_%.bbappend

```diff
@@ -1 +0,0 @@
-CFLAGS += "-fPIC"
```

Comment:
@rneilturner I have only compile tested, but it looks like leaving `CFLAGS += "-fPIC"` in this .bbappend enables PIC without needing your `compile-static-with-fPIC.patch`.  The option moves on the command line, but it's still there.

I prefer a separate .bbappend since it makes a clear distinction between our customization and the upstream recipe.  It also retains the history of why @Tmcg2 introduced the change.